cmd/nerdctl/image/image_remove_test.go

```diff
@@ -302,3 +305,58 @@ func TestRemove(t *testing.T) {
 
 	testCase.Run(t)
 }
+
+// TestIssue3016 tests https://github.com/containerd/nerdctl/issues/3016
+func TestIssue3016(t *testing.T) {
+	testCase := nerdtest.Setup()
+
+	const (
+		alpineImageName  = "alpine"
+		busyboxImageName = "busybox"
+		tagIDKey         = "tagID"
+	)
+
+	testCase.SubTests = []*test.Case{
+		{
+			Description: "Issue #3016 - Tags created using the short digest ids of container images cannot be deleted using the nerdctl rmi command.",
+			Require: test.Require(
+				test.Not(test.Windows),
+			),
+			Setup: func(data test.Data, helpers test.Helpers) {
+				helpers.Ensure("pull", alpineImageName)
```

Comment:
You should use `testutil.BusyboxImage` and `testutil.AlpineImage`

That will get you ghcr mirror versions instead of using docker hub which tend to error with 429.

Reply:
Thanks !! OK, I'll fix it.

Reply:
Fixed